docs/release-notes/master-docinfo.xml

```diff
@@ -7,7 +7,7 @@
         {ProductName} {DocInfoProductNumber} accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
     </para>
 	<para>
-		This document describes new features and improvements, known issues, and resolved issues for the {DocInfoProductName}, version 6.1.0.
+		This document describes new features and improvements, known issues, and resolved issues for the {DocInfoProductName}, version 6.2.0.
```

Comment:
@anarnold97 
This somewhat contradicts the first sentence of this abstract, which says 6.2 rather than 6.2.0, and the introduction, which says "These release notes cover all z-stream releases of MTA 6.2".  Maybe this line should be similar. Although I don't understand why there should be two identical pieces of text as an abstract and an introduction.